contrib/babelfishpg_tsql/src/linked_servers.c

```diff
@@ -801,7 +802,8 @@ linked_server_establish_connection(char *servername, LinkedServerProcess * lspro
 		}
 
 		/* fetch query timeout from the servername */
-		query_timeout = get_query_timeout_from_server_name(servername);
+		if(!isTesting)
```

Comment:
add a comment explaining why we are not fetching query timeout in case of testing connecting